pkg/selinux/setroubleshoot-client.js

```diff
@@ -111,7 +112,10 @@ client.init = function() {
           doText
           analysisId: plugin id. It can be used in org.fedoraproject.SetroubleshootFixit.run_fix()
           fixable: True when an alert is fixable by a plugin
-          reportBug: True when an alert should be reported to bugzilla
+          reportBug: True when an alert should be reported
+      firstSeen: when the alert was seen for the first time, iso8601 format is used - '%Y-%m-%dT%H:%M:%SZ'
+      lastSeen: when the alert was seen for the last time, iso8601 format is used - '%Y-%m-%dT%H:%M:%SZ'
+      level: "green", "yellow" or "red"
```

Comment:
> Displaying human readable strings is orthogonal to that, though, and I don't think we should leave those time stamps like this.

I changed it to use momentjs:
![screenshot from 2016-05-24 10-16-35](https://cloud.githubusercontent.com/assets/8711649/15497277/3bbc8a84-219a-11e6-8d29-59bcb3ab179c.png)